src/core/storage_api.tf

```diff
@@ -1,5 +1,5 @@
 module "storage_api" {
```

Comment:
I think we need to enable new configuration `use_legacy_defender_version = false` for `api` and `apireplica`

Reply:
It's in another [PR](https://github.com/pagopa/io-infra/pull/882)